src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -0,0 +1,135 @@
+import moment from 'moment'
+import React, { useState } from 'react'
+import { Button, Table } from 'react-bootstrap'
+import { SMALL_USER_CARD } from '../utils/SynapseConstants'
+import {
+  AccessApprovalSearchRequest,
+  AccessApprovalSearchSort,
+  AccessApprovalSortField,
+  SortDirection,
+} from '../utils/synapseTypes'
+import Typography from '../utils/typography/Typography'
+import IconSvg from './IconSvg'
+import UserCard from './UserCard'
+import { PRODUCTION_ENDPOINT_CONFIG } from '../utils/functions/getEndpoint'
+import { useSearchAccessApprovalsInfinite } from '../utils/hooks/SynapseAPI/dataaccess/useAccessApprovals'
+import { SynapseSpinner } from './LoadingScreen'
+
+export type AccessApprovalsTableProps = {
+  accessorId: string
+  accessRequirementId?: string
+}
+export const AccessApprovalsTable: React.FunctionComponent<
+  AccessApprovalsTableProps
+> = ({ accessorId, accessRequirementId }) => {
+  const [isDescending, setIsDescending] = useState(false)
+  const [sort, setSort] = useState<AccessApprovalSearchSort[]>()
+
+  const searchRequest: AccessApprovalSearchRequest = {
+    accessorId,
+    accessRequirementId,
+    sort,
+  }
+
+  const { data, hasNextPage, fetchNextPage, isFetching } =
+    useSearchAccessApprovalsInfinite(searchRequest)
+
+  const accessApprovals = data?.pages.flatMap(page => page.results) ?? []
+  const onSort = (field: AccessApprovalSortField, isDescending: boolean) => {
+    const direction: SortDirection = isDescending
+      ? ('DESC' as SortDirection)
+      : ('ASC' as SortDirection)
+    setSort([{ field, direction }])
+    setIsDescending(!isDescending)
+  }
+  return (
+    <div className="AccessApprovalsTable bootstrap-4-backport">
+      <Typography style={{ padding: '12px' }} variant="headline3">
+        Status in Access Requirements
+      </Typography>
+      <Table striped borderless bordered={false}>
+        <thead className="access-approval-header">
+          <tr>
+            <th className="thead-cell remove-border">AR ID</th>
+            <th className="thead-cell remove-border">
+              Access Requirement Name
+            </th>
+            <th className="thead-cell remove-border">Submitter</th>
+            <th className="thead-cell remove-border">Status</th>
+            <th className="thead-cell remove-border button-cell">
+              Modified Date
+              <button
+                style={{ float: 'right' }}
+                onClick={() =>
+                  onSort(AccessApprovalSortField.MODIFIED_ON, isDescending)
+                }
+              >
+                <IconSvg
+                  options={{
+                    icon: 'sortTwoTone',
+                  }}
+                />
```

Comment:
Let's use SortIcon in Sort.tsx, which has props/styling that indicate the current sort

Something like this (I don't think this will work perfectly):
```suggestion
                <SortIcon
                  active={sort.field === 'CREATED_ON'}
                  direction={
                    sort.field === 'CREATED_ON' ? sort.direction : 'DESC'
                  }
                />
```